lib/sidekiq/cli.rb

```diff
@@ -417,5 +417,5 @@ def rails_app?
   end
 end
 
-require "sidekiq/systemd"
+require "sidekiq/systemd" if Gem::Specification.find_all_by_name("sd_notify").any?
```

Comment:
Too specific. What if the user wants to supply their own version or use a forked gem? Just catch LoadError like usual?

Reply:
They can require the file manualy. 

The only requirement is that is should be named sd_notify.rb and SdNotify